core/data.go

```diff
@@ -40,6 +40,17 @@ type Blob struct {
 	Data          []byte
 }
 
+type BlobAuthHeader struct {
+	// Commitments
+	BlobCommitments `json:"commitments"`
+	// AccountID is the account that is paying for the blob to be stored
+	AccountID AccountID `json:"account_id"`
```

Comment:
Is this decided already (to be the eth address)?

Reply:
There's some uncertainty about how this will marry with the payments design, but I think it's okay to move forward with supporting a simple ECDSA-based authentication. We can swap this out pretty easily, imo.

Currently, the AccountID shoul be the ECDSA public key. I'll update the comment.